pkg/virt-controller/services/template.go

```diff
@@ -763,6 +763,8 @@ func newSidecarContainerRenderer(sidecarName string, vmiSpec *v1.VirtualMachineI
 		requestedHookSidecar.Image = os.Getenv(operatorutil.SidecarShimImageEnvName)
 	}
 
+	sidecarOpts = append(sidecarOpts, WithContainerName(sidecarName))
```

Comment:
Any reason not to add the `WithContainerName()` option when initializing `sidecarOpts`?